unity-renderer/Assets/DCLServices/WearablesCatalogService/WearableResponse.cs

```diff
@@ -188,7 +190,8 @@ public long GetMostRecentTransferTimestamp()
 
             public WearableItem ToWearableItem(string contentBaseUrl, string bundlesBaseUrl)
             {
-                var wearable = entity.ToWearableItem(contentBaseUrl, bundlesBaseUrl);
+                string individualId = individualData != null ? individualData[0].id : string.Empty;
+                var wearable = entity.ToWearableItem(contentBaseUrl, bundlesBaseUrl, amount, individualId);
```

Comment:
I would rather prefer overriding the id here than passing it as a parameter, because i think `firstIndividualId` adds noise in the entity process. The entity already has its own id:
```
string individualId = individualData != null ? individualData[0].id : string.Empty;
var wearable = entity.ToWearableItem(contentBaseUrl, bundlesBaseUrl, amount);

if (!string.IsNullOrEmpty(individualId))
    wearable.id = individualId;
```

What do you think?